lib/pg_query/version.rb

```diff
@@ -1,3 +1,3 @@
 module PgQuery
-  VERSION = '6.0.0'.freeze
+  VERSION = '6.1.0'.freeze
```

Comment:
The `Gemfile.lock` also includes a reference to the version, which should be updated with a simple `bundle`